rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -183,11 +183,46 @@ logstash_apt_keys: "{{ rpco_apt_gpg_keys }}"
 # The MAAS apt repo cannot be included in the integrated apt repository
 # due to the fact that the same package name/version is used for multiple
 # distributions, but the content and checksums are different.
-maas_apt_repos:
-  - repo: "deb {{ rpco_mirror_base_url }}/apt-mirror/independant/rax-maas-{{ ansible_distribution_release }}/ {{ rpc_release }}-{{ ansible_distribution_release }} main"
-    state: "present"
-    filename: "rax-maas"
-maas_apt_keys: "{{ rpco_apt_gpg_keys }}"
+maas_repos:
+  repo: "deb {{ rpco_mirror_base_url }}/apt-mirror/independant/rax-maas-{{ ansible_distribution_release }}/ {{ rpc_release }}-{{ ansible_distribution_release }} main"
+  state: "present"
+  filename: "rax-maas"
+
+maas_keys:
+  hash_id:  "{{ rpco_gpg_key_id }}"
+  url: "{{ rpco_gpg_key_location }}{{ rpco_gpg_key_name }}"
+  state: "present"
+
+# Set the maas_scheme
+# Note: (alextricity25) This has been changed to https because
+# SSL termination is done by default since Newton per https://review.openstack.org/#/c/277199/
+maas_scheme: https
+
+# Set the following to skip a specific check
+# horizon_local_check is disabled until https://github.com/rcbops/u-suk-dev/issues/781
+# is resolved
+maas_excluded_checks:
```

Comment:
group-vars for rpc-o are not honored by an external set of playbooks. Because these playbooks exist outside of RPC-O we'll need maas specific variables to become stand alone vars.